JSDemos/Demos/Toast/Overview/Angular/app/app.component.html

```diff
@@ -1,13 +1,16 @@
-<div id="toast">
+<div id="productList">
```

Comment:
I suggest using a kebab-case for class and id names.
Because we use this case style everywhere
https://github.com/DevExpress/devextreme-demos/blob/21_1/JSDemos/Demos/Button/Icons/Vue/App.vue#L89

Reply:
@onebalaban Did you decide not to do that?

Reply:
I thought that kebab-case is related only to Vue. I'll check other frameworks and will submit a PR.